lib/nimble_template/addons/variants/phoenix/web/bootstrap.ex

```diff
@@ -0,0 +1,117 @@
+defmodule NimbleTemplate.Addons.Phoenix.Web.Bootstrap do
+  @moduledoc false
+
+  use NimbleTemplate.Addons.Addon
+
+  @impl true
+  def do_apply(%Project{} = project, opts) do
+    project
+    |> edit_files(opts)
+    |> copy_files()
+  end
+
+  defp edit_files(%Project{} = project, opts) do
+    project
+    |> edit_assets_package()
+    |> edit_app_js(opts)
+    |> edit_app_scss(opts)
+    |> edit_vendor_index(opts)
+
+    project
+  end
+
+  defp copy_files(%Project{} = project) do
+    copy_bootstrap_vendor(project)
+
+    project
+  end
+
+  defp edit_assets_package(%Project{} = project) do
+    Generator.replace_content(
+      "assets/package.json",
+      """
+        "dependencies": {
+      """,
+      """
+        "dependencies": {
+          "@popperjs/core": "^2.11.2",
+          "bootstrap": "5.1.3",
+      """
+    )
+
+    project
+  end
+
+  def edit_app_js(project, %{with_nimble_js_addon: true}) do
```

Comment:
this method and the other ones, e.g. `edit_vendor_js` don't need to be public right?
```suggestion
  defp edit_app_js(project, %{with_nimble_js_addon: true}) do
```

Reply:
Good catch @longnd will update it

Reply:
Updated it in https://github.com/nimblehq/elixir-templates/pull/161/commits/650f2461c2fac990450aacf24ea8650cd1ac0454